pkg/install/install.go

```diff
@@ -487,3 +490,9 @@ func (i *Installer) deployARMTemplate(ctx context.Context, rg string, tName stri
 
 	return err
 }
+
+// addResourceProviderVersion sets the deploying resource provider version in the cluster document for deployment-tracking purposes.
+func (i *Installer) addResourceProviderVersion(ctx context.Context) error {
+	i.doc.OpenShiftCluster.Properties.ProvisionedBy = version.GitCommit
```

Comment:
this isn't guaranteed to be persisted - you need to explicitly make a PatchWithLease call here - copy the way of doing it from somewhere else

Reply:
Done -- had a bit of trouble finding how the tests worked (since it's a mutating call), but I managed to figure it out :)